app/controllers/charge_controller.js

```diff
@@ -183,9 +183,13 @@ module.exports = {
   checkCard: (req, res) => {
     const namespace = getNamespace(clsXrayConfig.nameSpaceName)
     const clsSegment = namespace.get(clsXrayConfig.segmentKeyName)
+    const normalisedCardNo = normalise.creditCard(req.body.cardNo)
+    if (normalisedCardNo && normalisedCardNo.length < 11) {
+      return res.json({ 'accepted': false, message: 'Card number is not the correct length' })
+    }
```

Comment:
I think it might be better to have this check inside the `checkCard` function in `card.js` and have a `return reject(new Error('Card number is not the correct length'))` in there — that way the existing code in this file for constructing the response would just work